README.md

```diff
@@ -22,35 +22,55 @@ A Very Good Infinite List Widget created by [Very Good Ventures][very_good_ventu
 
 ## Usage
 
-A basic `InfiniteList` requires two parameters:
+A basic `InfiniteList` requires four parameters:
 
-- `itemLoader` which is responsible for asynchronously fetching the content
-- `builder` which is responsible for returning a `Widget` given a specific item (result)
+- A list of `items` that are displayed using the `itemBuilder`.
+- An `itemBuilder` that is responsible for returning a widget for every element in `items`.
```

Comment:
thoughts on further aligning with `ListView.builder` and accepting an `itemCount` and an `itemBuilder` which provides the index and context rather than `items`? This way the `InfiniteList` widget itself does not need to have a generic `T` because it doesn't care about the type of item.